app/gist/new/route.js

```diff
@@ -7,7 +7,7 @@ export default GistRoute.extend({
   model () {
     this.store.unloadAll('gistFile');
 
-    var model = this.store.createRecord('gist', {description: 'New Twiddle'});
+    var model = this.store.createRecord('gist', {description: 'New Twiddle '});
```

Comment:
Yes, I know. Sorry. Maybe instead of adding a space to the value you can add a space to a computed property based on value and show that?